openapi-v3.yaml

```diff
@@ -223,6 +247,9 @@ paths:
           name: order_by
           schema:
             type: string
+            anyOf:
+            - enum: [id, created_at, -id, -created_at]
+            - type: "string"
```

Comment:
We may want to consider removing or adding to this list. I believe any field on Job can be used for `order_by`, but these are the two fields I most commonly see.